applications/video_streaming/video_streaming_client/cpp/CMakeLists.txt

```diff
@@ -124,4 +124,45 @@ install(
     DESTINATION bin
 )
 
+# Enable testing
+include(CTest)
+
+# Testing infrastructure
+if(BUILD_TESTING)
```

Comment:
Could you please document in the README how to 1) build and 2) run the tests?

Seems like there is an undocumented manual prerequisite for testing that both apps must be launched to get test results. Instead, could we consolidate in a test script so that a single test can run startup, functionality, and post-processing without manual intervention? This would allow us to automate the test coverage in the future to guard against regressions.